### PR TITLE
2.2.0-0.2.1: [FEAT]: CLBOSS UI

### PR DESCRIPTION
**Pull request by johngribbin**
- Adds a new `Plugins` tile to the homepage 
- Adds a new plugins dashboard, with a wallet toggle, to allow users to manage their plugins on a per wallet basis.
- Adds a dashboard for the `CLBOSS` plugin that exposes some commands.
- Users can refresh the CLBOSS status as well as inform the plugin to ignore onchain funds for a certain number of hours (defaulting to 24)
- Users can also update fine grained permissions on a per channel basis. For example - do not close a given channel, regardless of any performance metrics that are being tracked by the plugin. 

Screenshot 2024-08-03 at 8 15 10 AM
Screenshot 2024-08-03 at 8 15 22 AM

### CODE REVIEW

**Comment by lnbc1QWFyb24** [comment]
Have not had a good look through the code yet, but did take it for a spin locally and found a couple of small issues:

- Navigating to plugins route when no wallet has been collected results in an infinite spinner:
Screenshot 2024-08-04 at 10 46 46 AM

- Navigating to the plugins route with a node connected that does not have the CLBoss plugin installed results in an infinite spinner:
Screenshot 2024-08-04 at 10 52 32 AM

- An issue from a previous PR, but worth noting, when connecting a node, it flashes the red bitcoind is not synced when connecting which ideally it would not do

**Comment by johngribbin** [comment]
> Have not had a good look through the code yet, but did take it for a spin locally and found a couple of small issues:
> 
> * Navigating to plugins route when no wallet has been collected results in an infinite spinner:
> 
> Screenshot 2024-08-04 at 10 46 46 AM
> * Navigating to the plugins route with a node connected that does not have the CLBoss plugin installed results in an infinite spinner:
> 
> Screenshot 2024-08-04 at 10 52 32 AM
> * An issue from a previous PR, but worth noting, when connecting a node, it flashes the red bitcoind is not synced when connecting which ideally it would not do

ah sorry about that. Should be fixed now. Should only see the spinner when trying to make a connection to the node.

**Review by lnbc1QWFyb24 — APPROVED** [review]
Nice one!

File: src/lib/constants.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import list from '$lib/icons/list.js'
 import settingsOutline from '$lib/icons/settings-outline.js'
 import wallet from '$lib/icons/wallet.js'
 import forward from '$lib/icons/forward.js'
+import terminal from './icons/terminal.js'
 import type { Network } from './@types/payments.js'
```
Comment (slater82) [inline src/lib/constants.ts:11]: BTC address to receive payments from crypto in bitcoin bc1q9w9vctw8h2u5fzrgmgat83k9yfk6hy3qpsq5v7

Comment (slater82) [reply src/lib/constants.ts:11]: Lincensed.

**Review by slater82 — APPROVED** [review]

**Pull request merged**